recommended-homework-solution/assets/css/style.css

```diff
@@ -0,0 +1,321 @@
+  border: 1px solid #dddddd;
+}
+
+/* portfolio page */
+
+.work {
+  position: relative;
+  float: left;
+  width: 274px;
+  margin: 20px 0 25px;
+  overflow: auto;
+}
+
+.work:nth-child(even) {
+  margin-right: 40px;
+}
+
+.work img {
+  width: 100%;
+  border: 0 none;
+  opacity: 0.8;
+}
+
+.work h3 {
+  position: absolute;
+  bottom: 20px;
+  width: 100%;
+  padding: 15px;
+  margin-bottom: 0;
+  font-weight: 300;
+  line-height: 30px;
+  color: #ffffff;
+  text-align: center;
+  background: #4aaaa5;
+  border-bottom: 0;
+}
+
+.auth-image {
+  float: left;
+  width: 200px;
+  height: auto;
+  margin-top: 10px;
+  margin-right: 25px;
+}
+
+/* contact page */
+
+#contact-form ul {
+  margin-bottom: 20px;
+}
+
+#contact-form li {
+  margin-bottom: 10px;
+}
+
+label,
+input[type=text],
+input[type=email],
+textarea {
+  display: block;
+  width: 100%;
+}
+
+input[type=text],
+input[type=email],
+textarea {
+  height: 35px;
+  padding: 0 10px;
+  font-size: 14px;
+  border: 1px solid #dddddd;
+}
+
+textarea {
+  height: 200px;
+}
+
+input[type=submit] {
+  padding: 10px 30px;
+  font-size: 18px;
+  color: #ffffff;
+  cursor: pointer;
+  background: #4aaaa5;
+  border: 0 none;
+}
+
+@media screen and (max-width: 980px) {
+
+  #main-container {
+    float: left;
+    width: 100%;
+    padding-top: 130px;
+    min-height: calc(100vh - 70px);
+  }
+
+  .main-section {
+    float: left;
+    width: 75%;
+    max-width: 960px;
+    padding: 30px;
+    margin: 0 0 40px;
+    background: #ffffff;
+    border: 1px solid #dddddd;
+  }
+
+
+}
+
+@media screen and (max-width: 768px) {
+  #main-container {
+    width: 100%;
+    padding-top: 130px;
+    min-height: calc(100vh - 70px);
+  }
+
+  .main-section {
+    float: none;
+    width: 100%;
+    height: auto;
+    background:#ffffff;
+    border: 1px solid #dddddd;
+  }
+
+  .auth-image {
+    float: left;
+    width: 300px;
+    height: auto;
+    margin-top: 10px;
+    margin-right: 25px;
+  }
+
+
+}
+
+@media screen and (max-width: 640px) {
+  #main-container {
+    padding-top: 0px;
+  }
+
+  #logo {
+    float: center;
+    width: 250px;
+    height: 90px;
+    font-family: Georgia, Times, "Times New Roman", serif;
+    font-size: 30px;
+    font-weight: 700;
+    line-height: 90px;
+    color: #ffffff;
+    text-align: center;
+    text-decoration: none;
+    background: #4aaaa5;
+  }
+
+  nav {
+    clear: none;
+    float: left;
+    margin-top: 25px;
+  }
+
+  .main-section {
+    float: top;
+    width: 100%;
+    padding: 30px;
+    margin: 0 0 40px;
+    background: #ffffff;
+    border: 1px solid #dddddd;
+  }
+
+  .auth-image {
+    float: left;
+    width: 100%;
+    height: auto;
+    margin-top: 10px;
+    margin-right: 25px;
+  }
+
+  #masthead {
+    position: static;
+    z-index: 99;
+    width: 100%;
+    color: #ffffff;
+    background: #ffffff;
+    border-bottom: 2px solid #cccccc;
+  }
+
+}
+
+/* Struggling with how to make nav bar go under the header for 640px */
```

Comment:
By header do you mean, "Welcome"? If so, it seems that the navbar you are using has its own media query and when it is resized it is going into the dropdown menu.